cluster-up/cluster/kind-k8s-sriov-1.17.0/provider.sh

```diff
@@ -2,7 +2,16 @@
 
 set -e
 
-export CLUSTER_NAME="sriov"
+DEFAULT_CLUSTER_NAME="sriov"
+DEFAULT_HOST_PORT=5000
```

Comment:
I don't know much about it, so I could be wrong, but is it safe enough to give a hard coded port, could it be take already?
Is there a way to allocate a random free port?

Reply:
Its totally safe, as long as there isn't other port that holds it.
(else cluster-up would simply fails, which is ok)

We already have port 5000 hardcoded, now you can have two clusters, each will have its own port, chosen according the cluster name.
`make cluster-sync` will be also automatic aware of it, since its propagated in the node as well, and taken from there.
I don't think we need to over engineer here.

Its in order to be able to run two cluster locally, switch easily, and in the future debug even problems that might occur if two clusters run on the same time on CI.
Atm CI is safer than locally, because it runs in DinD.

Reply:
Thanks for explaining.